ecommerce/views/v0/__init__.py

```diff
@@ -465,9 +472,8 @@ def get(self, request):
         except ObjectDoesNotExist:
             return HttpResponse("No basket")
 
-        if "no_checkout" in checkout_payload:
+        if "no_checkout" or "country_blocked" in checkout_payload:
```

Comment:
This needs to be two separate "in checkout_payload"s so the if doesn't short circuit early. (If the user can check out normally, you get a 500 error from this line.)